### ISSUE TITLE
Unable to reproduce Flowers-> Birds experiment

### ISSUE THREAD

**Issue by shivamsaboo17** [rahafaljundi/MAS#5]
Hi I am trying to reproduce results for MAS for flowers-> birds using alexnet (imagenet pretrained) and using same hyperparameters mentioned in the paper. Here are the results that I am getting (image size 256):

With MAS:
flowers: ~85%
birds: ~40%

Without MAS:
flowers: ~25%
birds: ~42% 

I believe that MAS works because without it the flowers category is degrading. But the accuracy on birds is less. Have you used any preprocessing or data augmentation while training?

Also I tried MAS (and EWC) on class based incremental learning where the task ID is not given during test/train time and all classes must be predicted. The performance of MAS drops drastically in this setup. This was also demonstrated in this paper - (https://arxiv.org/pdf/1904.07734.pdf). Can you provide any insights why there is so much difference in the task and class based setup?
